### DISCUSSION TITLE
"Could not start watcher. Make sure you have required chokidar. Error output: sh: line 0: exec: : not found\n"

### DISCUSSION THREAD

**Discussion by emyasnikov** [spatie/file-system-watcher#10] [Q&A]
Hi,

I have a problem to start the watcher process.

Every time I start process exception returns following:
"Could not start watcher. Make sure you have required chokidar. Error output: sh: line 0: exec: : not found\n".

But chokidar is installed with npm and should be accessible, at least through node_modules.

The project runs with Laravel 9 and has Vue 2 based frontend, so I have no idea where to require chokidar instead.

I also tried to "fix" the issue using [Fixed getWatchProcess #2](https://github.com/spatie/file-system-watcher/pull/2) without success.

Any ideas how to fix it?

**Comment by emre037** [comment]
post this on issues then

**Comment by AndMe2** [comment]
Ok checked around for quite a while but after finding this command: type -a node
which resulted in:
node is /usr/bin/node
node is /bin/node
I changed the first part of the line in the example configuration from
command=env PATH="/usr/local/bin"
to 
command=env PATH="/usr/bin"

Running ubuntu 20.04.
Now it is working, yaY. Hope this helps.

Full line is:
command=env PATH="/usr/bin" /usr/bin/php /var/www/your_ project/artisan watch:scan
